packages/graph/hash_graph/lib/graph/src/api/rest/api_resource.rs

```diff
@@ -9,7 +9,7 @@ use crate::GraphPool;
 /// through a `Router`, making it explicitly clear we want to provide `OpenApi` specification as
 /// documentation for the routes.
 pub(super) trait RoutedResource: utoipa::OpenApi {
-    fn routes<P: GraphPool>() -> Router;
+    fn routes<P: StorePool + Send + 'static>() -> Router;
```

Comment:
`GraphPool` was an alias to `StorePool` with `StorePool::Store` bound to all the `Read` implementations, which is not needed anymore. `Send + 'static` is required for `axum`